### PR TITLE
fix(v2/number): cast to int directly if no decimal point is found to avoid wrong values because of precision loss

### PR DESCRIPTION
**Pull request by chiragjn**
## JIRA Ticket Number

JIRA TICKET:  ML-2315

## Description of change
Cast to int directly if no decimal point is found to avoid wrong values because of precision loss.

## Checklist (OPTIONAL):

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ner_v2/detectors/numeral/number/standard_number_detector.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 import collections
 import os
+import decimal
```
Comment (haptik-deployment) [inline ner_v2/detectors/numeral/number/standard_number_detector.py:6]: F401 'decimal' imported but unused

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Review by tanaya-b — APPROVED** [review]

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**